sklearn/linear_model/tests/test_logistic.py

```diff
@@ -90,6 +91,18 @@ def test_predict_3_classes():
     check_predictions(LogisticRegression(C=10), X_sp, Y2)
 
 
+def test_lr_liblinear_warning():
+    n_samples, n_features = iris.data.shape
+    target = iris.target_names[iris.target]
+
+    for LR in [LogisticRegression(solver='liblinear', n_jobs=2),
+               LogisticRegressionCV(solver='liblinear', n_jobs=2)]:
```

Comment:
I remember `n_jobs` for `LogisticRegressionCV` does have an effect and lead to usage of multiple CPU cores even when the solver is `liblinear`, but not for `LogisticRegression`. In the `LogisticRegressionCV` case, I thought it's just multiple instances/processes of `LogisticRegression` running at the same time, but each instance itself still uses a single core. This doesn't seem to conflict with that `n_jobs` doesn't work `LogisticRegression` with `liblinear` solver. Please correct me if I am wrong.

I thought @glemaitre would be more familiar with the conventions used in `scikit-learn`, so I suggested him to make my requests more conformant with the conventions.